### PR TITLE
feat(pkger): adds markdown chart support

### PR DESCRIPTION
**Pull request by dearyhud**
Closes #

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: pkger/models.go [hunk 1]
```diff
@@ -790,13 +792,17 @@ func (c chart) properties() influxdb.ViewProperties {
 			Note:              c.Note,
 			ShowNoteWhenEmpty: c.NoteOnEmpty,
 		}
-	case chartKindHeatMap:
+	case chartKindMarkdown:
+		return influxdb.MarkdownViewProperties{
+			Type: influxdb.ViewPropertyTypeMarkdown,
+			Note: c.Note,
```
Comment (jsteenb2) [inline pkger/models.go]: 👏

File: pkger/models.go [hunk 2]
```diff
@@ -933,6 +944,19 @@ func (c chart) validBaseProps() []failure {
 	return fails
 }
 
+func (c chart) validNote() []failure {
```
Comment (jsteenb2) [inline pkger/models.go]: nit: you can do this inline, don't think anything else will be reusing this

File: pkger/models.go [hunk 3]
```diff
@@ -773,12 +774,13 @@ func (c chart) properties() influxdb.ViewProperties {
 			Note:              c.Note,
 			ShowNoteWhenEmpty: c.NoteOnEmpty,
 		}
-	case chartKindScatter:
+	case chartKindHeatMap:
 		ia := c.Axes.influxAxes()
-		return influxdb.ScatterViewProperties{
```
Comment (dearyhud) [inline pkger/models.go]: I just alphabetized things here. didn't introduce anything new besides the reading in the Markdown props.

File: pkger/models.go (hunk 2 above)
Comment (jsteenb2) [reply pkger/models.go]: also, is a `""` a valid note for markdown? not sure what our validation is for markdowns 🤷‍♂

File: pkger/models.go [hunk 4]
```diff
@@ -863,11 +869,17 @@ func (c chart) properties() influxdb.ViewProperties {
 func (c chart) validProperties() []failure {
 	var fails []failure
 
+	if c.Kind == chartKindMarkdown {
+		// at the time of writing, there's nothing to validate for markdown types
+		return fails
```
Comment (jsteenb2) [inline pkger/models.go]: nit: `return nil` here, and move the fails var down to closer to where it is being used,  above the validatorFns var

**Pull request merged**